src/components/ec-inline-input-field/components/copy/copy.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div
+    class="ec-inline-input-field-copy"
+    data-test="ec-inline-input-field-copy"
+  >
+    <span class="ec-inline-input-field-copy__text">
+      {{ value }}
+    </span>
+
+    <button
+      type="button"
+      class="ec-inline-input-field-copy__action"
+      data-test="ec-inline-input-field-copy__action"
+      @click="copy"
+    >
+      <ec-icon
+        class="ec-inline-input-field-copy__icon"
+        name="simple-copy"
+        :size="16"
+      />
+    </button>
+  </div>
+</template>
+
+<script>
+import EcIcon from '../../../ec-icon';
+
+const clipboardCopy = require('clipboard-copy');
+
+export default {
+  name: 'EcInlineInputFieldCopy',
+  components: { EcIcon },
+  props: {
+    value: {
+      default: '',
+      type: String,
+    },
+  },
+  methods: {
+    copy() {
+      clipboardCopy(this.value);
```

Comment:
What design said about successful/unsuccessful copy action? This call returns resolve promise if copy was done or rejected promise if copy command failed.

Reply:
Hey Milos, for this ticket will be only the copy functionality. Will create a new subtask/pr shortly where I'll be tackling the success/error state after consolidating design. @matallo is aware of this. Will add this comment in the pr description.